code/controllers/subsystem/blackbox.dm

```diff
@@ -32,10 +32,17 @@ SUBSYSTEM_DEF(blackbox)
 
 	CheckPlayerCount()
 
-	if(CONFIG_GET(flag/use_exp_tracking))
-		if((triggertime < 0) || (world.time > (triggertime +3000)))	//subsystem fires once at roundstart then once every 10 minutes. a 5 min check skips the first fire. The <0 is midnight rollover check
+	if((triggertime < 0) || (world.time > (triggertime +3000)))	//subsystem fires once at roundstart then once every 10 minutes. a 5 min check skips the first fire. The <0 is midnight rollover check
+		if(CONFIG_GET(flag/use_exp_tracking))
 			update_exp(10,FALSE)
 
+		if(SSdbcore.Connect())
```

Comment:
i need you to comment your code changes with 
`// VOID EDIT`
thanks <3

Reply:
(if you're unsure why, im planning on setting up a pr mirror bot so this will help to prevent future conflicts)

Reply:
should i also do that for removing code?

Reply:
yeah, just comment it out with `VOID EDIT`  on the line before, or the same line

Reply:
Done!